server/members/management/commands/send_verification_reminders.py

```diff
@@ -0,0 +1,231 @@
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--all", action="store_true", help="Send reminder to all users"
+        )
+        parser.add_argument(
+            "--username", type=str, help="SWECC Interview Website Username"
+        )
+        parser.add_argument(
+            "--dry-run", action="store_true", help="Dry run, do not send emails"
+        )
+        parser.add_argument(
+            "--preview", action="store_true", help="Preview email content"
+        )
+
+    def is_verified(self, user):
+        return user.groups.filter(name="is_verified").exists()
+
+    def get_target_users(self, options):
+        if options["username"]:
+            users = User.objects.filter(username=options["username"])
+            if not users.exists():
+                return None, f"User with username {options['username']} not found"
+            return list(users), None
+
+        if options["all"]:
+            users = [user for user in User.objects.all() if not self.is_verified(user)]
+            if not users:
+                return None, "No unverified users found"
+            return users, None
+
+        return None, "Either --all or --username must be specified"
+
+    def send_reminder_email(self, user):
+        html_content = email_template(user)
+        subject = "SWECC Account Verification Required"
+
+        if not user.email:
+            return False, f"⚠ User {user.username} has no email address"
+
+        try:
+            send_email(
+                from_email=SENDER_EMAIL,
+                to_email=user.email,
+                subject=subject,
+                html_content=html_content,
+            )
+            return True, f"✓ Sent reminder to {user.email}"
+        except Exception as e:
+            return False, f"✗ Failed to send reminder to {user.email}: {str(e)}"
+
+    def preview_email(self, user):
+        html_content = email_template(user)
+        self.stdout.write(
+            self.style.SUCCESS(f"\nPreview of email for {user.username}:\n")
+        )
+        self.stdout.write(html_content)
+        self.stdout.write("\n")
+
+    def handle(self, *args, **options):
+        pp = (
+            lambda user: f"Username: {user.username}, Email: {user.email}, Discord ID: {user.discord_id}"
+        )
+
+        users, error = self.get_target_users(options)
+        if error:
+            self.stdout.write(self.style.ERROR(error))
+            return
+
+        self.stdout.write(
+            self.style.SUCCESS(
+                f"Found {len(users)} {'user' if len(users) == 1 else 'users'} to send reminders to"
+            )
+        )
+
+        for user in users:
+            self.stdout.write(
+                self.style.SUCCESS(pp(user) + f", Verified: {self.is_verified(user)}")
+            )
+
+        if options.get("preview"):
+            self.preview_email(users[0])
+            return
+
+        if options["dry_run"]:
```

Comment:
If `dry_run` isn't in options, this would throw a `KeyError`. Expected behavior should be that if `dry_run` isn't specified we assume it to be false. Here's one way of checking:
```suggestion
        if options.get("dry_run") is not None:
```

Reply:
are you sure?

Reply:
yeah `store_true` makes them default to false unless provided

Reply:
Ah ok, just make sure to access either one with the same pattern then, just for consistency (i.e., either use `.get` for both or indexing for both)